bin/run.js

```diff
@@ -50,6 +62,74 @@ module.exports = function run( files, options ) {
 	QUnit.on( "runEnd", function setExitCode( data ) {
 		if ( data.testCounts.failed ) {
 			process.exitCode = 1;
+		} else {
+			process.exitCode = 0;
+		}
+	} );
+};
+
+run.restart = function( args ) {
+	if ( QUnit.config.queue.length ) {
+		console.log( "Finishing current test and restarting..." );
+	} else {
+		console.log( "Restarting..." );
+	}
+
+	run.abort( () => run.apply( null, args ) );
```

Comment:
Spread/rest are not supported in Node 4 unfortunately 😞 https://kangax.github.io/compat-table/es6/#test-rest_parameters